### PR TITLE
Slopen checker (incl. need-report conclusion type)

### PR DESCRIPTION
**Pull request by afjlambert**
Next to contact-conclusion, permit-free, needs permit we now have (Meldingsplicht) need-report.
Noting too fancy, just reused what we already have.
Used a new checker from production sttr builder.

url: /bouwwerk-slopen

TODO:
- [x] we need a report outcome text component
- [x] Checks are fake, we need legal to enter the right questions
- [x] we need a new QuestionAlert type
- [x] do we need to implement more Matamo events?
- [x] we need more tests 
- [x] we need to remove the new QuestionAlert type
- [x] update conclusion texts
- [x] update intro page text
- [ ] we need to do more manual testing
- [ ] do we need to redirect old OLO-check urls to the imtr-flow?

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [ ] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [ ] you added the necessary documentation (either in the markdown files or inline)
- [ ] you added the necessary automated tests

### CODE REVIEW

**Comment by robinpiets** [comment]
I've added the "need report" Alert

File: packages/imtr-client/src/models/checker.ts [hunk 1]
```diff
@@ -312,4 +313,35 @@ export default class Checker {
     }
     return question || null;
   }
+
+  /**
+   * needReportOutcome (copy-paste from needContactExit)
+   * TODO: Refactor into one function? only `imtrOutcomes` is used
```
Comment (afjlambert) [inline packages/imtr-client/src/models/checker.ts]: Yes, let's refactor before removing the Draft status. 👍

Comment (robinpiets) [reply packages/imtr-client/src/models/checker.ts]: This is done

**Comment by robinpiets** [comment]
Closing this PR so @afjlambert can review

**Pull request closed without merging**